enterprise/migrations/0197_auto_20231130_2239.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.23 on 2023-11-30 22:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0196_backfill_sso_marked_authorized'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='enterprisecustomerssoconfiguration',
+            name='marked_authorized',
+            field=models.BooleanField(default=False, help_text='Whether admin has indicated the service provider metadata was uploaded.'),
```

Comment:
huh I would have thought you needed to specify the blank/null values in the migration... I guess that's an ORM level thing and not a DB level thing? 🤔 I should really know this 😂

Reply:
uhh scratch that.. you do it need it... what is this migration doing?

Reply:
I see this is setting the default value- I think this can go on the original migration file? 0195